resque-round-robin.gemspec

```diff
@@ -10,9 +10,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "resque", "~> 1.19"
   gem.add_dependency "resque-dynamic-queues"
-  # Newer versions of redis-rb break many different things for us
-  # Test carefully when relaxing this restriction
-  gem.add_dependency "redis", "<= 4.5"
+  gem.add_dependency "redis", "<= 4.7.1"
```

Comment:
We know that versions less than 4.7 will cause problems so should this be `~> 4.7.1` instead?

Reply:
Correct, fixed 👍